mobile-app/app/contexts/DeFiScanContext.tsx

```diff
@@ -58,3 +65,8 @@ function getTxURLByNetwork (network: EnvironmentNetwork, txid: string, rawtx?: s
 
   return baseUrl
 }
+
+function getBlocksURLByNetwork (network: EnvironmentNetwork, blockCount: number): string {
+  const baseUrl = `https://defiscan.live/blocks/${blockCount}${getNetworkParams(network)}`
```

Comment:
Can we store the URL as a constant on this file?

Reply:
URL is used on getBlocksURL and getTxURL

Reply:
I will do that in next PR for tokens where we will use a common method to get the url